SwaprHQ/swapr-dapp
# [SWA-129][FEAT] - Add Scroll chain support

## Fixes: [SWA-129](https://linear.app/swaprhq/issue/SWA-129/add-scroll-chain)

# Description

* Update the pre-selected output currency for Gnosis, to be WETH

# Visual Evidence
![image](https://github.com/SwaprHQ/swapr-dapp/assets/21271189/875f9750-2b1c-4b77-9772-685c3cb57483)


# How to test the changes

1) Pull this branch
2) Run the project locally
3) Go to Swapr dapp page
4) Connect your wallet
5) Connect to Scroll chain and try to check balances and/or do a swap. The token quotes should appear through Sushi v3
